integrated_channels/blackboard/client.py

```diff
@@ -464,3 +529,21 @@ def _get_or_create_integrated_grade_column(self, bb_course_id):
                 )
 
         return grade_column_id
+
+    def create_integration_content_for_course(self, bb_course_id, course_content_child_data):
+        """
+        Helper method to generate the default blackboard course content page with the integration custom information.
+        """
+        course_content_data = {
+            'title': 'edX Integration',
+            'position': 0,
+            "contentHandler": {"id": "resource/x-bb-folder"}
+        }
+        content_url = self.generate_create_course_content_url(bb_course_id)
+
+        content_resp = self._post(content_url, course_content_data)
+        bb_content_id = content_resp.json().get('id')
+
+        content_child_url = self.generate_create_course_content_child_url(bb_course_id, bb_content_id)
+        self._post(content_child_url, course_content_child_data)
+        return bb_content_id,
```

Comment:
does the comma here indicate we want to return a tuple? is this necessary?

Reply:
also isn't it nice to have the status code and text returned here as well maybe in addition to this info to keep our pattern of preserving that info through out the code?

Reply:
I thiiiiink I was originally returning status/body, thus the comma haha.

Reply:
There's a point here about logging, which is at the heart of why I made the switch originally, we're creating a content object and a child to that content object. Which ID do we log? Do we even need to log anything? I can follow our patterns and return the status code and body of the response right above, but then if we log the ID from the response, we're logging the content's child's ID, which could potentially lead to confusion while debugging. I'm flexible here, what do you think?

Reply:
yeah as a simple rule if we keep that status text, code tuple to be always the return value, it will help with consistency as well as reporting. having said that, you can have more info returned than just code and body, like this case, so could be benefit from an output like (status_code, status_text, bb_content_id) ? as long as the method is clear about what it returns I thought having the two pieces of info always first, then followed by other needed info could be useful.

also, depending on how you want to handle failures for this response, it may make that work easier too if we keep track of the status and text from the post method above this line